### ISSUE TITLE
Debian/Ubuntu packaged version missing

### ISSUE THREAD

**Issue by brendangregg** [iovisor/bpftrace#489]
Using the packages:

```
# bpftrace --version
bpftrace 
```

Should say the version number, but doesn't.

steps:

```
sudo apt-get install software-properties-common
sudo add-apt-repository ppa:canonical-kernel-team/kernel-tools-edge
sudo apt-get update && sudo apt-get install bpftrace
bpftrace --version
```

**Comment by caringi** [comment]
The same happens in Fedora...

```
# bpftrace --version
bpftrace 
```

The reason is that the current `--version` implementation relies solely on running `git describe --abbrev=4 --dirty --always --tags` and distros do the builds from tarballs instead of git repos.

**Comment by mmarchini** [comment]
Fixed by #513, but we need to update upstream packages before closing this.
